src/status_im2/contexts/wallet/events.cljs

```diff
@@ -259,6 +267,32 @@
           [:wallet/request-collectibles
            {:start-at-index start-at-index}]])]})))
 
+(rf/reg-event-fx :wallet/get-collectible-details
+ (fn [_ [collectible-id]]
+   (let [request-id               0
+         collectible-id-converted (cske/transform-keys csk/->PascalCaseKeyword collectible-id)
+         request-params           [request-id [collectible-id-converted]]]
+     {:json-rpc/call [{:method   "wallet_getCollectiblesDetailsAsync"
+                       :params   request-params
+                       :on-error (fn [error]
+                                   (log/error "failed to request collectible"
+                                              {:event  :wallet/get-collectible-details
+                                               :error  error
+                                               :params request-params}))}]})))
```

Comment:
Shouldn't the `rpc-call` has an `:on-success` ?

Reply:
@OmarBasem, this call doesn't return anything immediately, so I just omitted it. Meaningful results will come from signal later.
Also in the code of `:json-rpc/call` having `:on-success` is not obligatory, it goes under `when` clause:
```
(when on-success
               (let [result (if js-response
                              (.-result response-js)
                              ...
```

Reply:
> this call doesn't return anything immediately

I am curious, so when the data finally arrives how is it being handled?